content/04.results.md

```diff
@@ -28,21 +28,21 @@ This reverses the primary direction of information flow shown in Panel A.
 As a baseline, we evaluated prediction of mutation status from gene expression data across several different gene sets.
 Past work has evaluated mutation prediction for the top 50 most mutated genes in TCGA [@doi:10.1186/s13059-020-02021-3], and we sought to extend this to a broader list of gene sets.
 We compared a set of cancer-related genes from Vogelstein et al. 2013 [@doi:10.1126/science.1235122] with a set of random genes having equal size, and a set of the most mutated genes in TCGA having equal size.
-For all gene sets, we used only the set of TCGA samples for which both gene expression and somatic mutation data exists, resulting in a total of 9,074 samples from all 33 cancer types (Figure {@fig:expression_gene_sets}A).
+For all gene sets, we used only the set of TCGA samples for which both gene expression and somatic mutation data exists, resulting in a total of 9,074 samples from all 33 cancer types.
 This set of samples was further filtered for each target gene to cancer types containing at least 15 mutated samples and at least 5% of samples mutated for that cancer type.
 We then evaluated the performance for each target gene in each of the three gene sets.
 
-Figure {@fig:expression_gene_sets}B summarizes performance for all genes in each gene set.
+Figure {@fig:expression_gene_sets}A summarizes performance for all genes in each gene set.
 Genes from the Vogelstein et al. set were more predictable than randomly chosen genes or those selected by total mutation count.
```

Comment:
```suggestion
Genes from the Vogelstein et al. set were more predictable than randomly chosen genes or those selected by total mutation count (Figure {@fig:expression_gene_sets}A).
```
A stylistic thing, but I think it would be worth considering putting the figure's take-away in the sentence (as opposed to a description of what the figure shows) and then removing the sentence before this.

Reply:
This might come up each time you have a figure referenced (looking below).